assets/training/finetune_acft_hf_nlp/components_v2/finetune/chat_completion/spec.yaml

```diff
@@ -1,14 +1,14 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 name: oss_chat_completion_finetune
-version: 0.0.22
+version: 0.0.23
```

Comment:
Update data import components as well

Reply:
data import used different environment "azureml://registries/azureml/environments/acft-hf-nlp-data-import"

Reply:
Still new version need to be released